openslides_backend/action/actions/user/create.py

```diff
@@ -15,13 +18,13 @@
 from ...util.register import register_action
 from ...util.typing import ActionResultElement
 from ..meeting_user.mixin import CheckLockOutPermissionMixin
-from .create_update_permissions_mixin import CreateUpdatePermissionsMixin
 from .password_mixins import SetPasswordMixin
 from .user_mixins import LimitOfUserMixin, UserMixin, UsernameMixin, check_gender_helper
 
 
 @register_action("user.create")
 class UserCreate(
+    UserMixin,
```

Comment:
Then try just `**transfer_field_list`

Reply:
I tried but it seems that a class attribute from another package is only visible by giving its class name.
Which makes sense because the `UserMixin` was also part of the inheritance hierarchy before moving it directly here.

Reply:
I also tried moving the attribute initialization to the __init__ method, but that would defeat the purpose of class attributes and leads to many other problems.

Reply:
Alright, then leave it